test/jdk/java/lang/Thread/virtual/stress/Skynet.java

```diff
@@ -25,14 +25,25 @@
  * @test
```

Comment:
Would you mind adding "id=default" to this test? If a test has several `@test` tags, some with an identifier, and some without, then it can be tricky to understand test selection and exclusion.